app/test_helpers.go

```diff
@@ -54,6 +54,18 @@ type SetupOptions struct {
 	AppOpts            types.AppOptions
 }
 
+func DefaultOptions() SetupOptions {
+	return SetupOptions{
+		Logger:             log.NewNopLogger(),
+		DB:                 dbm.NewMemDB(),
+		InvCheckPeriod:     5,
+		HomePath:           DefaultNodeHome,
+		SkipUpgradeHeights: map[int64]bool{},
+		EncConfig:          MakeEncodingConfig(),
+		AppOpts:            EmptyAppOptions{},
+	}
+}
```

Comment:
As mentioned above, I think these are all app test utils and should live together in a single package.

Reply:
We can also handle migrating these helpers in a separate pull request but we should prepare the `testutil` package if you think this would be a good idea.